buildSrc/src/main/java/Config.kt

```diff
@@ -1,7 +1,7 @@
 import java.math.BigDecimal
 
 object Config {
-    val kotlinVersion = "1.4.10"
+    val kotlinVersion = "1.4.30"
```

Comment:
```suggestion
    val kotlinVersion = "1.4.10"
```

lets rollback to avoid conflicts, i am likely downgrading to v1.3.x